### PR TITLE
move data bags to default location, add chef-zero default for easier …

### PR DESCRIPTION
**Pull request by rmoriz**
…data bag handling in dev

(as discussed in https://gist.github.com/rmoriz/13f1b87026a3a5ef2c15 )

### CODE REVIEW

File: chefignore [hunk 1]
```diff
@@ -0,0 +1,9 @@
+*/.chef
```
Comment (joschi) [inline chefignore]: Shouldn't this file be named `.chefignore` (https://docs.chef.io/chef_repo.html#chefignore-files) or is the leading dot optional?

Comment (rmoriz) [inline chefignore]: afaik documentation error at docs.chef.io. They changed it in ~2010 and still need to clean it up…

Comment (joschi) [inline chefignore]: :+1:

**Comment by jamescott** [comment]
https://github.com/chef/chef-docs/pull/524

**Pull request merged**